jina/importer.py

```diff
@@ -26,62 +92,32 @@ def import_classes(namespace: str, targets=None,
 
     import os, re
 
-    if namespace == 'jina.executors':
-        import_type = 'ExecutorType'
-        if import_once and IMPORTED.executors:
-            return
-    elif namespace == 'jina.drivers':
-        import_type = 'DriverType'
-        if import_once and IMPORTED.drivers:
-            return
-    elif namespace == 'jina.hub':
-        import_type = 'ExecutorType'
-        if import_once and IMPORTED.hub:
-            return
-    else:
+    _namespace2type = {
+        'jina.executors': 'ExecutorType',
+        'jina.drivers': 'DriverType',
+        'jina.hub': 'ExecutorType'
+    }
+    _import_type = _namespace2type.get(namespace)
+    if _import_type is None:
         raise TypeError(f'namespace: {namespace} is unrecognized')
 
-    from setuptools import find_packages
-    import pkgutil
-    from pkgutil import iter_modules
+    _targets = _get_targets(targets)
+    if _targets is None:
+        raise TypeError(f'target must be a set, but received {targets!r}')
```

Comment:
This can never happen, since `_get_targets` will never return `None`. Regardless, I'd rename `_get_targets` into `_parse_targets` and do any error handling there.

Reply:
As `targets` is no longer used after #1759, I've removed this part.